ext/intl/formatter/formatter_class.c

```diff
@@ -25,6 +25,7 @@
 #include "formatter_attr.h"
 
 #include <zend_exceptions.h>
```

Comment:
Shouldn't this rather be
```suggestion
#include "Zend/zend_exceptions.h"
```